### PR TITLE
Fix issue: #337

### PR DESCRIPTION
**Pull request by webcarrot**
Fix: https://github.com/millermedeiros/esformatter/issues/337

### CODE REVIEW

**Comment by millermedeiros** [comment]
can you please add tests and a descriptive commit message?

**Comment by millermedeiros** [comment]
for the tests you just need to include some extra lines on the https://github.com/millermedeiros/esformatter/blob/master/test/compare/default/class_declaration-in.js and https://github.com/millermedeiros/esformatter/blob/master/test/compare/default/class_declaration-out.js and run `npm test` to see if all tests passes.

**Comment by webcarrot** [comment]
I added tests, array expressions and improve multi line support. (sorry for my english)

File: test/compare/default/class_declaration-in.js [hunk 1]
```diff
@@ -33,3 +33,15 @@ return this.properties;
...
+// Multi line declaration
+class
+Foo
+extends
+Bar
+{
+}
+// The value to be extended can be produced by an arbitrary expression.
+class Foo extends BarNamespace.Bar{}
+class Foo extends BarNamespace [ 'Bar' ] . Bar [ "Bar" ] ( 'bar' ) . Bar ( "bar", "bar" ){}
```
Comment (millermedeiros) [inline test/compare/default/class_declaration-in.js]: I'm going to merge your patch but I think I'm going to ignore this edge case (ignore the fact that input might contain weird white spaces in between the properties).. since it's complex and not really part of the `ClassDeclaration` (should be handled by `MemberExpression` and `CallExpression` instead)
